tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/tracing/CustomTracingTestCase.java

```diff
@@ -44,16 +45,18 @@
     @Test
     public void testAddCustomSpanToSystemTrace() throws Exception {
         final String resourceName = "resourceOne";
-        final String span1Position = FILE_NAME + ":21:5";
-        final String span3Position = FILE_NAME + ":32:19";
-        final String span5Position = FILE_NAME + ":45:20";
+        final String entrypointPosition = FILE_NAME + ":22:5";
+        final String span1Position = FILE_NAME + ":22:5";
```

Comment:
I think its ok to rename this to `resourceFunctionPosition` and keep only one variable.
```suggestion
        final String resourceFunctionPosition = FILE_NAME + ":22:5";
```

Reply:
Please fix in all the relevant places.

Reply:
FIxed with 3a5fad4b8ee95545ae441daea7a28e24052367f6